### PR TITLE
Setup nextjs app [STORM-1076]

### PR DESCRIPTION
**Pull request by niklas-lumio-aiven**
- Installed @aivenio/aquairum as the design-system
- Removed not needed code
- Set react strict mode to false
- To make aquarium work needed to define esmExternals as false

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,35 @@
-{{PROJECT_NAME}}
+nextjs-netlify
 ======================
-This is a template repository for creating open source repositories at Aiven.
+Recipe library built with Next.js, PostgreSQL, Redis, OpenSearch and hosted with Netlify. Recipe data is from https://www.kaggle.com/datasets/thedevastator/better-recipes-for-a-better-life.
 
 Overview
 ========
 
+Recipe library is a Next.js application that uses PostgreSQL as it's database. Optionally it utilizes Redis for caching favorite recipe statistics and OpenSearch for providing advanced search support for recipes.
```
Comment (blomqma) [inline README.md:8]: `... its database`*

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "my-app",
```
Comment (blomqma) [inline package.json:2]: nit: The app could be named something different, like `aiven-nextjs-netlify` etc.

File: src/pages/_document.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <Html lang='en'>
+      <Head />
+      <body>
+        <Main />
+        <NextScript />
+      </body>
+    </Html>
+  );
+}
```
Comment (blomqma) [inline src/pages/_document.tsx:13]: nit: Missing newline

File: package.json (hunk 2 above)
Comment (niklas-lumio-aiven) [reply package.json:2]: Fixed

File: src/pages/_document.tsx (hunk 3 above)
Comment (niklas-lumio-aiven) [reply src/pages/_document.tsx:13]: Fixed

**Comment by niklas-lumio-aiven** [comment]
Thanks for the review. Fixed the comments.

**Review by blomqma — APPROVED** [review]

**Pull request merged**